src/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+  display: flex;
+}
+   
+.stars_block {
+  display: flex;
+  flex-direction: column;
+  height: 96px;
+  width: 96px;

```

Comment:
do not hardcode - it is unnecessary here
just set `margin-right: 4px;` for `.stars__star`